Kernel/System/DB.pm

```diff
@@ -920,7 +920,11 @@ sub ListTables {
 
     my @Tables;
     while ( my ($Table) = $Self->FetchrowArray() ) {
-        push @Tables, lc $Table;
+        if ( $Param{Original} ) {
+            push @Tables, $Table;
+        } else {
```

Comment:
The coding standard is the 'else' is on a new line. Could you check whether CodePolicy automatically fixes this?